src/universal-dapp.js

```diff
@@ -334,7 +334,7 @@ UniversalDApp.prototype.getCallButton = function (args) {
 
   if (lookupOnly) {
     $contractProperty.addClass('constant')
-    call()
```

Comment:
you can't remove this . this function get the return value of a call just after the instance is added